Builds/levelization/results/ordering.txt

```diff
@@ -135,6 +135,7 @@ test.csf > ripple.protocol
 test.csf > test.jtx
 test.json > ripple.beast
 test.json > ripple.json
+test.json > ripple.rpc
```

Comment:
Adding lines to the ordering file is great, as long as it follows the existing rules (which this does). It would be a problem if the line was added to `loops.txt`, or if the dependency didn't "fit" the existing order.